docs/web/authentication.md

```diff
@@ -368,6 +368,9 @@ Is it possible to generate a token from command line which can be used to
 authenticate in the name of the given user. This way no need to store passwords
 in text files. For more information [see](#personal-access-token).
 
+The location of the password can be configured by the `CC_PASS_FILE`
```

Comment:
```suggestion
The location of the password file can be configured by the `CC_PASS_FILE`
```

Reply:
Please extend the documentation that this environment variable can be used to setup different credential files to login to the same server with a different user.